### PR TITLE
Group version fixes

### PR DESCRIPTION
**Pull request by mshitrit**
Fixing leftovers of some group version fixes I've missed (mainly regarding webhooks) at the original PR.

### CODE REVIEW

File: bundle/manifests/hasno-setup-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -51,9 +51,9 @@ metadata:
         }
       ]
     capabilities: Basic Install
+    operatorframework.io/suggested-namespace: ha-sno
     operators.operatorframework.io/builder: operator-sdk-v1.7.1+git
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
-    operatorframework.io/suggested-namespace: ha-sno
```
Comment (slintes) [inline bundle/manifests/hasno-setup-operator.clusterserviceversion.yaml:56]: is this line removed on purpose?

Comment (mshitrit) [reply bundle/manifests/hasno-setup-operator.clusterserviceversion.yaml:56]: Initially I've added this line manually, so it was deleted every time I've generated the bundle.
I've recently updated the manifest - as a result it was generated two lines up (that's why it seems deleted).

I'm planing to Push the manifest change in a different PR, but the location change of this line as a result of bundle generate leaked in to this PR (my apologizes ).

Comment (slintes) [reply bundle/manifests/hasno-setup-operator.clusterserviceversion.yaml:56]: Oh, I missed that it was just moved, sorry. Need more ☕ 😉

**Review by slintes — APPROVED** [review]

**Pull request merged**